site/_docs/history.md

```diff
@@ -31,6 +31,11 @@ Downloads are available on the
 ## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.23.0">1.23.0</a> / under development
 {: #v1-23-0}
 
+Compatibility: This release is tested on Linux, macOS, Microsoft Windows;
+using Oracle JDK 8, 9, 10, 11, 12, 13, 14 and OpenJDK 8, 9, 10, 11, 12, 13, 14;
```

Comment:
Please remove 9, 10, 11, 12, 13 from the list. We test only LTS (8, 11) and EA (14) JDK versions.
I'm not sure we test with Oracle JDK.

Reply:
I test on all of those JDKs.